functions/onprem/orborus/orborus.go

```diff
@@ -949,6 +951,13 @@ func stopWorker(containername string) error {
 func initializeImages() {
 	ctx := context.Background()
 
```

Comment:
Why don't you have an environment variable that signifies that the user wants to use tenzir?